utam-compiler/src/test/java/utam/compiler/grammar/UtamMethodAction_Tests.java

```diff
@@ -282,4 +285,20 @@ public void testGetParameterTypesForCustomAction() {
     assertThat(new Custom("customMethod", VOID, parameters).getParametersTypes(),
         is(containsInAnyOrder(PrimitiveType.BOOLEAN, SELECTOR)));
   }
+
+  @Test
+  public void testWaitWithIncorrectElementThrows() {
+    TranslationContext context = TestUtilities.getTestTranslationContext();
+    context.setElement(new Document());
+    new UtamElement(ELEMENT_NAME, "clickable", getListSelector()).testTraverse(context);
+    // document element does not have waitFor API
+    expectThrows(
```

Comment:
I think it's important when we're unit testing that a method throws an exception, that it's throwing the _right_ exception. Usually checking the message of the exception is good enough for this, but I know it's helped me track down several issues when refactoring.